get-access-token.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -e
+
+tokenUrl=$(cat /usr/oidc/credentials.json | jq -r '.tokenUrl')
```

Comment:
Doesn't make a practical difference, but this can be redirected instead of using cat:
```suggestion
tokenUrl=$(jq -r '.tokenUrl' < /usr/oidc/credentials.json)
```